GoogleChromeLabs/ps-analysis-tool
# Feature: Generate complete report from extension.

## Description
Previously when a report is downloaded from the extension it used to show the landing page only. To keep feature parity with the CLI, the report downloaded from the extension will be the exact same copy of the CLI-Dashboard. The downloaded report will have a sidebar to navigate through the table.

## Relevant Technical Choices



## Testing Instructions
- Clone this branch.
- In the terminal run `npm run ext:build`
- Now open any site in chrome and open the PSAT extension.
- Once download button is available. Click on it to download the report
- You should be able to navigate through frames.

## Additional Information:



## Screenshot/Screencast



---

## Checklist





Fixes #738
